### DISCUSSION TITLE
Set wire:dirty on filepond file upload?

### DISCUSSION THREAD

**Discussion by cgunnels** [livewire/livewire#7980] [1. Help] [answered]
How can I set the form to wire:dirty when I upload a file using filepond?

**Comment by joshhanley** [comment, marked as answer]
@cgunnels `wire:dirty` is only for properties that are in the browser that haven't been sent to the server yet. But file uploads trigger a request to the server straight away, so I don't think `wire:dirty` will work in that scenario.

**Reply by cgunnels** [reply]
Ok thanks and good to know. Is there a way to tell livewire that the form is "dirty" when a file has been uploaded?

**Reply by joshhanley** [reply]
@cgunnels you would have to detect it manually, as Livewire has no way of know what the initial property values are vs when there are changes (except for when they are sitting in the front end waiting for a request to happen).
